docs/guides/vault-admin-token.md

```diff
@@ -0,0 +1,22 @@
+---
+subcategory: ""
+page_title: "Create a Vault cluster and admin token - HCP Provider"
+description: |-
+    An example of creating a Vault cluster and admin token.
+---
+
+# Create a new Vault cluster and an admin token
+
+Once you have an HVN, HCP Vault enables you to quickly deploy a Vault Enterprise cluster in AWS across a variety of environments while offloading the operations burden to the SRE experts at HashiCorp.
+The cluster's admin token grants its bearer administrator access to the Vault cluster. It expires after 6 hours.
```

Comment:
I'm not sure how we expect the token to be used downstream of HCL like this, but I'm wondering if some more info here would be helpful (particularly regarding how the token expiration/refresh works). For example, if the token is being used elsewhere in Terraform, I don't think this 6 hour expiration is particularly actionable for the user, because if they `terraform apply` again after 6 hours, the token will be refreshed automatically.